MAINTAINERS

```diff
@@ -2,3 +2,4 @@ Thomas Stromberg <tstromberg@google.com> (@tstromberg)
 Anders F Björklund <anders.f.bjorklund@gmail.com> (@afbjorklund)
 Sharif Elgamal <selgamal@google.com> (@sharifelgamal)
 Medya Ghazizadeh <medya@google.com> (@medyagh)
+Jose Donizetti <jdbjunior@gmail.com> (@josedonizetti)
```

Comment:
Do you mind removing this file altogether? It is apparently unused, and was replaced with the OWNERS file.